aerospike/assets/dashboards/overview.json

```diff
@@ -1,1283 +1,960 @@
+            "definition": {
+                "type": "note",
+                "content": "IO",
+                "background_color": "gray",
+                "font_size": "18",
+                "text_align": "center",
+                "show_tick": true,
+                "tick_pos": "50%",
+                "tick_edge": "bottom"
+            },
+            "layout": {
+                "x": 41,
+                "y": 49,
+                "width": 120,
+                "height": 8
+            }
+        },
+        {
+            "id": 31,
+            "definition": {
+                "type": "note",
+                "content": "Disk",
+                "background_color": "gray",
+                "font_size": "18",
+                "text_align": "center",
+                "show_tick": true,
+                "tick_pos": "50%",
+                "tick_edge": "bottom"
+            },
+            "layout": {
+                "x": 41,
+                "y": 79,
+                "width": 120,
+                "height": 8
+            }
+        },
+        {
+            "id": 32,
+            "definition": {
+                "type": "note",
+                "content": "Memory",
+                "background_color": "gray",
+                "font_size": "18",
+                "text_align": "center",
+                "show_tick": true,
+                "tick_pos": "50%",
+                "tick_edge": "bottom"
+            },
+            "layout": {
+                "x": 41,
+                "y": 129,
+                "width": 120,
+                "height": 8
+            }
+        },
+        {
+            "id": 33,
+            "definition": {
+                "type": "note",
+                "content": "XDR",
+                "background_color": "gray",
+                "font_size": "18",
+                "text_align": "center",
+                "show_tick": true,
+                "tick_pos": "50%",
+                "tick_edge": "bottom"
+            },
+            "layout": {
+                "x": 41,
+                "y": 159,
+                "width": 120,
+                "height": 8
+            }
+        }
+    ],
+    "template_variables": [
         {
-            "add_timeframe": true,
-            "auto_refresh": false,
-            "bgcolor": "gray",
-            "board_id": "4iq-sen-4xc",
-            "font_size": "18",
-            "height": 8,
-            "html": "XDR",
-            "isShared": false,
-            "refresh_every": 30000,
-            "scaleFactor": 1,
-            "text_align": "center",
-            "tick": true,
-            "tick_edge": "bottom",
-            "tick_pos": "50%",
-            "title": false,
-            "type": "note",
-            "width": 120,
-            "x": 41,
-            "y": 159
+            "name": "scope",
+            "default": "*",
+            "prefix": null
         }
-    ]
+    ],
+    "layout_type": "free",
+    "is_read_only": true,
+    "notify_list": [],
+    "id": 30300
```

Comment:
Having an id isn't an issue for these, right?

Reply:
Nope, the frontend actually generates widget ids if one does not exist; otherwise it uses the id provided by the backend.